doc/usage.rst

```diff
@@ -25,20 +25,20 @@ The ``--format`` option for the output format. Supported formats are ``txt`` (de
 
 NOTE: the output for the following formats are generated in accordance with schemas
 
-* ``checkstyle` follows the common `"checkstyle" XML schema </doc/schemas/fix/checkstyle.xsd>`_
-* ``json` follows the `own JSON schema </doc/schemas/fix/schema.json>`_
-* ``junit`` follows the `JUnit XML schema from Jenkins </doc/schemas/fix/junit-10.xsd>`_
-* ``xml`` follows the `own XML schema </doc/schemas/fix/xml.xsd>`_
+* ``checkstyle`` follows the common ``"checkstyle" XML schema </doc/schemas/fix/checkstyle.xsd>``
+* ``json`` follows the ``own JSON schema </doc/schemas/fix/schema.json>``
+* ``junit`` follows the ``JUnit XML schema from Jenkins </doc/schemas/fix/junit-10.xsd>``
+* ``xml`` follows the ``own XML schema </doc/schemas/fix/xml.xsd>``
```

Comment:
The formatting was correct here to create links: `` `text <URL>` ``. Now the syntax is rendered literally, please revert.